pkg/drivers/kic/oci/oci.go

```diff
@@ -257,6 +257,25 @@ func createContainer(ociBin string, image string, opts ...createOpt) error {
 	return nil
 }
 
+// StartContainer starts a container with "docker/podman start"
+func StartContainer(ociBin string, container string) error {
+	// construct the actual docker start argv
+	args := []string{"start"}
+
+	// to run nested container from privileged container in podman https://bugzilla.redhat.com/show_bug.cgi?id=1687713
```

Comment:
ad a comment here and say why only linux, so ppl dont revert it to all later.

Reply:
Maybe we should wrap it in some "IsPodmanRemote" function globally even

Reply:
yeah we could do that

Reply:
comment added